### DISCUSSION TITLE
Resizing with Decklink HDMI out.  Resize algorithm selection?

### DISCUSSION THREAD

**Discussion by alatteri** [CESNET/UltraGrid#368] [Q&A]
Hello,

Our input is often 2K DCI (2048x1080) SDI captured via Decklink.  When outputting this content via Decklink HDMI, the sides get chopped out as the HDMI signal is standard HD (1920x1080).   I've played with using a resize filter on output, but the quality was very poor and unusable.  Is there a way to specify a particular resize algorithm that might provide better quality?  Or is there a different way to achieve this?

Thanks,
Alan

**Comment by mikekoetter** [comment]
Are all of your endpoints 1080?
I've used the Capture Filter to resize the incoming signal with good results

**Reply by alatteri** [reply]
Hi Mike,

No, some endpoints  use Decklink SDI out which does 2K DCI and they can display that.

**Reply by MartinPulec** [reply]
Hi Alan, actually (and coincidentally) I've added that option to the _resize_ capture filter (or postprocess) just yesterday if you think this filter. You can try it with `resize:<size>:algo=cubic`.

**Reply by alatteri** [reply]
Thank you Martin.  I will take a look, hopefully in a week or so.  Been really busy.  Thanks again.
